Makefile

```diff
@@ -1,10 +1,14 @@
-default: build_10 build_12 build_14
+all: 10-stretch-slim 12-stretch-slim 14-stretch-slim
```

Comment:
Does `stretch-slim` give us access to Debian-based builds? Is that the difference between what was the default before?